pdf_build_src/process_markdowns.py

```diff
@@ -141,6 +142,194 @@ def modify_changelog():
         file.writelines(data)
 
 
+# Number of chars maximal in one line approximated from a line of the PDF
+NB_CHARS_LINE_PDF = 100
```

Comment:
still feels a bit weird to me to not determine this automatically ... but if it works for now, fine by me to leave it. Somebody else can automate determining this number later.

Reply:
https://github.com/bids-standard/bids-specification/pull/560#discussion_r467073656